code/script.js

```diff
@@ -1,9 +1,81 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+const faqHeaderOne = document.getElementById('faqHeaderOne');
```

Comment:
Regarding this whole file I wanted to mention one thing :

For now, this code is ok as we didn't yet learn how to write it in more universal way, nonetheless I want you to know that in general it's not OK to define 18 variables for each header and content and attach same repeated logic to all of them.
Soon we will learn tools and thanks to them we will write this whole logic below in 3-5 lines ;)